README.md

```diff
@@ -173,7 +173,8 @@ Here are the common options:
             anchor: new google.maps.Point(0, 32)
         },
         label: function(point) {
-            return '<div class="map-custom-label">' + point.id + '</div>';
+            // a function that return the text value of the label
+            return `${point.data.location.name}`;
```

Comment:
Why using string literals here?
Either just return `point.data.location.name` or `String(point.data.location.name)` if you want to cast it to string (be careful of casting null, arrays, objects, functions, etc...)

Reply:
You're right. Just return point.data.location.name